pyodide_build/build_env.py

```diff
@@ -19,8 +19,7 @@
 from pyodide_build.recipe import load_all_recipes
 
 RUST_BUILD_PRELUDE = """
-rustup toolchain install ${RUST_TOOLCHAIN} && rustup default ${RUST_TOOLCHAIN}
-rustup target add wasm32-unknown-emscripten --toolchain ${RUST_TOOLCHAIN}
+rustup default ${RUST_TOOLCHAIN}
```

Comment:
I suppose we could also do this in `_ensure_rust_toolchain()` and remove `RUST_BUILD_PRELUDE` entirely.

Reply:
Then , will `pyodide build-recipes-no-deps` work?